app/javascript/confirmation_modal.js

```diff
@@ -0,0 +1,33 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
can we convert this to a stimulus controller instead? we have a few modal controllers there already.